test/test_autograd.py

```diff
@@ -2936,20 +2936,6 @@ def test_igammac(self):
         gradcheck(torch.igamma, (s, x))
         gradgradcheck(torch.igamma, (s, x))
 
-    def test_chain_matmul(self):
```

Comment:
Let's also change calls to chain_matmul in the code base to linalg.multi_dot, like here:

https://github.com/pytorch/pytorch/blob/7297556d5d9561a1c0a4b96ad068c13e05ff9a5c/test/test_autograd.py

and 

https://github.com/pytorch/pytorch/blob/6dc763df3098485f158eeed2b9621cffaa229669/torch/nn/utils/spectral_norm.py